### PR TITLE
YASP: Update Simplified Chinese

### PR DESCRIPTION
**Pull request by JohnsonRan**

### CODE REVIEW

File: res/values-zh-rCN/yaap_strings.xml [hunk 1]
```diff
@@ -49,54 +51,54 @@
...
     <string name="network_traffic_arrow_summary">显示网速箭头</string>
-    <string name="network_traffic_type">选择显示方式</string>
+    <string name="network_traffic_type">显示方式</string>
     <string name="show_network_traffic_combined">合并（上传+下载）</string>
     <string name="show_network_traffic_dynamic">动态</string>
     <string name="show_network_traffic_up">上传</string>
     <string name="show_network_traffic_down">下载</string>
     <string name="show_network_traffic_all">同时显示</string>
-    <string name="network_traffic_font_size_title">仅显示单向时字体大小</string>
+    <string name="network_traffic_font_size_title">单向显示32时字体大小</string>
```
Comment (ALKALiKong233) [inline res/values-zh-rCN/yaap_strings.xml]: Ig this is a typo mistake that u mistakenly added '32' here? @JohnsonRan

Comment (ALKALiKong233) [reply res/values-zh-rCN/yaap_strings.xml]: others looks fine btw

**Review by ALKALiKong233 — APPROVED** [review]
fine now 👍

**Review by idoybh — APPROVED** [review]
LGTM

**Comment by idoybh** [comment]
thank you @JohnsonRan and @ALKALiKong233

**Pull request merged**